make-release.sh

```diff
@@ -38,6 +38,9 @@ sed -i -e "s/cloudshell-image:v\([0-9\.]\+\)/cloudshell-image:${NEW_VERSION}/g"
 sed -i -e "s/cloudshell_git_branch=v\([0-9\.]\+\)/cloudshell_git_branch=${NEW_VERSION}/g" ${REPO_ROOT}/docs/index.html;
 sed -i -e "s/cloudshell-image:v\([0-9\.]\+\)/cloudshell-image:${NEW_VERSION}/g" ${REPO_ROOT}/docs/index.html;
 
+# update tags for load generator image
+find "${REPO_ROOT}/kubernetes-config" -name '*.yaml' -exec sed -i -e "s/:latest/:${NEW_VERSION}/g" {};
```

Comment:
This looks like it's missing the \ at the end. Otherwise it gives an error

Also, it would probably fit better with the other line under `# temporarily pin manifests to :$NEW_VERSION`